discovery-provider/tests/utils.py

```diff
@@ -226,12 +226,14 @@ def populate_mock_db(db, entities, block_offset=0):
             )
             session.add(user_listening_history)
 
-        for i, indexing_checkpoint_meta in enumerate(indexing_checkpoints):
-            indexing_checkpoint = models.IndexingCheckpoints(
-                tablename=indexing_checkpoint_meta.get("tablename", None),
-                last_checkpoint=indexing_checkpoint_meta.get("last_checkpoint", 0),
-            )
-            session.add(indexing_checkpoint)
+        if indexing_checkpoints:
```

Comment:
Future work -- we should probably move this truncation into a test fixture just to be done at the start & teardown of each test instead, but LGTM for now

Reply:
Added to linear task for cleaning up dp tasks.